app/controllers/queue_items_controller.rb

```diff
@@ -0,0 +1,48 @@
+class QueueItemsController < ApplicationController
+
+  def index
+    @queue_items = current_user.queue_items
+  end
+
+  def create
+    q = build_queue_item
```

Comment:
I would recommend to avoid 1 letter variables like this. You want to aim for a code that is more intention revealing rather than making it ambiguous. In your head, q might be what you want, but for other people its mind-taxing. And your future-you :). Make sure you are always explicit, other people will appreciate it when working with you.